[docs] Update guide for eas-builds to include expo-updates part

satya164 opened the pull request:
Update docs to incorporate changes from https://github.com/expo/expo-cli/pull/2587

dsokal reviewed (approved):
Thanks, please rebase this before merging.

satya164 committed:
[docs] Update guide for eas-builds to include expo-updates part
Update docs to incorporate changes from https://github.com/expo/expo-cli/pull/2587
